EOF
message
github.com/<user>/errors.(*TestFmtWithStack).Run
	<path>/github.com/<user>/errors/format_test.go:<line>
github.com/<user>/errors.TestFormatWithStack
	<path>/github.com/<user>/errors/format_test.go:<line>
testing.tRunner
runtime.goexit
github.com/<user>/errors.(*TestFmtWithStack).Run
	<path>/github.com/<user>/errors/format_test.go:<line>
github.com/<user>/errors.TestFormatWithStack
	<path>/github.com/<user>/errors/format_test.go:<line>
testing.tRunner
runtime.goexit
github.com/<user>/errors.(*TestFmtWithStack).Run
	<path>/github.com/<user>/errors/format_test.go:<line>
github.com/<user>/errors.TestFormatWithStack
	<path>/github.com/<user>/errors/format_test.go:<line>
testing.tRunner
runtime.goexit